docs/ALERTER_CONFIGS.md

```diff
@@ -64,7 +64,7 @@ alerter:
     type: email
     description: "primary email alerter"
     options:
-      to: daniel.selans@gmail.com
+      to: someone.important@gmail.com
```

Comment:
figured you might not want someone copy pasting this example and emailing you...